ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -880,6 +880,52 @@ def in_transit_encryption_verification():
     )
 
 
+def disable_in_transit_encryption():
+    """
+    Disable in_transit encryption.
+    """
+    ocp_obj = StorageCluster(
+        resource_name=constants.DEFAULT_CLUSTERNAME,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+    )
+
+    patch_cmd = '[{ "op": "replace", "path": "/spec/network/connections/encryption/enabled", "value": false }]'
+    log.info("patching storageclass to disable in-transit encryption.")
+    ocp_obj.patch(params=patch_cmd, format_type="json")
+    ocp_obj.wait_for_phase("Progressing", timeout=60)
+    verify_storage_cluster()
+    return True
+
+
+def enable_in_transit_encryption():
+    """
+    Enable in-transit encryption
+    """
+    ocp_obj = StorageCluster(
+        resource_name=constants.DEFAULT_CLUSTERNAME,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+    )
+
+    patch_cmd = '[{ "op": "replace", "path": "/spec/network/connections/encryption/enabled", "value": true }]'
+    log.info("patching storageclass to enable in-transit encryption.")
+    ocp_obj.patch(params=patch_cmd, format_type="json")
+    ocp_obj.wait_for_phase("Progressing", timeout=60)
+    verify_storage_cluster()
+    return True
+
+
+def verify_in_transit_encryption_keys_exists():
+    """
+    verify that "ms_client_mode", "ms_cluster_mode", "ms_service_mode" keys are exists
+    in 'ceph config dump output'.
+    """
+    try:
+        in_transit_encryption_verification()
+    except Exception:
```

Comment:
`in_transit_encryption_verification()` raises ValueError. Please except this exception instead of `Exception`, as that is too wide.
Even better IMO, would be to modify in_transit_encryption_verification() so it will return True in case the verification succeeds and return False in case the verification fails.
That way, you won't need to think of which exception to catch here.
The exception then could be raised in the test case level with `assert`